src/hooks/useDrag.tsx

```diff
@@ -494,12 +600,14 @@ export const onDelete = (
       const sections = deleteElement(frontMatter.sections, indexToDelete)
       const newErrorSections = deleteElement(errors.sections, indexToDelete)
       const newDisplaySections = deleteElement(displaySections, indexToDelete)
+      const newTextcardErrors = deleteElement(errors.textcards, indexToDelete)
```

Comment:
i'm assuming this should be `subindexToDelete` - could you check downstream if the deletion is done properly also?

Reply:
See above! This is for deleting a section, so the first layer of errors in `errors.textcards` should be deleted